src/plugins/mer/merplugin.cpp

```diff
@@ -172,7 +172,7 @@ bool MerPlugin::initialize(const QStringList &arguments, QString *errorString)
         ActionManager::actionContainer(Core::Constants::M_TOOLS);
 
     ActionContainer *menu = ActionManager::createMenu(Constants::MER_TOOLS_MENU);
-    menu->menu()->setTitle(tr("&Sailfish OS"));
+    menu->menu()->setTitle(Sdk::osVariant());
```

Comment:
I wonder if it assigns some shortcut key automatically

Reply:
It not will be automatically, add "&" befor